backend/app/store/engine/bolt.go

```diff
@@ -617,6 +645,80 @@ func (b *BoltDB) flagBucket(tx *bolt.Tx, flag Flag) (bkt *bolt.Bucket, err error
 	return bkt, nil
 }
 
+// checkUserDetail returns requested userDetail
+func (b *BoltDB) checkUserDetail(req UserDetailRequest) (val string) {
+	key := req.UserID
+	if key == "" {
+		return "" // return nothing in case UserID is not set
+	}
+
+	bdb, err := b.db(req.Locator.SiteID)
+	if err != nil {
```

Comment:
what the reason to eat errors here? I get why checkSometing may do it, but get should return the real error and not to mask it with empty string result